File: php/bintang_beda_simbol.php
<?php

$arr = ['*','#','$','%'];
$lenArr = count($arr);

$simbol = ['*','*','*','*'];

for ($i=0; $i<=$lenArr-1; $i++){	// 1,22,333,4444
	for ($j=0; $j<=$i; $j++)		// 1,12,123,1234
	{
	echo "{$arr[$i]} {$arr[$j]}<br>";
	

}}
// endfor
?>

File: python/bintangBedaSimbol.py
arr = ['*','#','$','%','@','$','*','#','$','%','@','$'];
arrLen = len(arr)

for i in range(0, arrLen-1):
	for j in range(0, i):
		print(arr[i] + arr[j], end='')
	print('')


